test/chunker-rabin.spec.js

```diff
@@ -70,11 +70,10 @@ describe('chunker: rabin', function () {
     const KiB256 = 262144
     let file = Buffer.concat([rawFile, Buffer.from('hello')])
     const opts = {
+      ...defaultOptions,
       minChunkSize: KiB256 / 3,
       avgChunkSize: KiB256,
-      maxChunkSize: KiB256 + (KiB256 / 2),
-      window: 16,
-      polynomial: '0x3DF305DFB2A805'
+      maxChunkSize: KiB256 + (KiB256 / 2)
     }
 
     const chunks = await all(chunker([file], opts))
```

Comment:
Should we add a test to check that the output of JS rabin impl is the same as native rabin impl?

Reply:
The rabin tests run in node and the browser, one of them [tests the size of chunks emitted](https://github.com/ipfs/js-ipfs-unixfs-importer/blob/master/test/chunker-rabin.spec.js#L23-L50) so I thought that was sufficient.

I was adding interop tests for more comprehensive testing but then @hugomrdias compiled the C version to WASM in [rabin-wasm](https://github.com/hugomrdias/rabin-wasm) came along so may wait for that to be ready.

Reply:
Interop tests are at ipfs/interop#74 - though as ever the build is failing due to unrelated tests that depend on networking.